addons/mail/static/src/models/composer/composer.js

```diff
@@ -1031,10 +1165,12 @@ function factory(dependencies) {
         isLog: attr({
             default: false,
         }),
+        isProcessingQueue: attr({
+            default: false
+        }),
         /**
          * Determines whether a post_message request is currently pending.
          */
-        isPostingMessage: attr(),
```

Comment:
Should not remove a field in stable, either keep it with no value ever assigned or with a related to `isSendingMessages`

Reply:
Should we put `deprecated` inside the doc string ?